font/bmfont.ksy

```diff
@@ -0,0 +1,155 @@
+meta:
+  id: bmfont
+  title: BMFont binary file
+  file-extension: fnt
+  license: CC0-1.0
+  endian: le
+  bit-endian: le
+doc-ref: https://www.angelcode.com/products/bmfont/doc/file_format.html  
+seq:
+  - id: magic
+    contents: ['BMF', 3]
+  - id: info
+    type: info
+  - id: common
+    type: common 
+  - id: pages
+    type: pages
+  - id: chars
+    type: chars
+  - id: kernings
+    type: kernings
+    if: not _io.eof
+types:
+  info:
+    seq:
+      - id: magic
+        contents: [1]
+      - id: block_size
+        type: s4
+      - id: size
+        type: s2        
+      - id: smooth
```

Comment:
Should the flags be packed into a dedicated subtype?

Reply:
if I'm not mistaken this will require an intermediate id to access the fields instead of directly accessing, maybe I'm missing some pros?